packages/indy-vdr/src/anoncreds/IndyVdrAnonCredsRegistry.ts

```diff
@@ -106,57 +110,41 @@ export class IndyVdrAnonCredsRegistry implements AnonCredsRegistry {
 
   public async registerSchema(
     agentContext: AgentContext,
-    options: IndyVdrRegisterSchemaOptions
+    options: RegisterSchemaOptions
   ): Promise<RegisterSchemaReturn> {
-    if (!options.options.didIndyNamespace) {
-      return {
-        schemaMetadata: {},
-        registrationMetadata: {},
-        schemaState: {
-          reason: 'no didIndyNamespace defined in the options. didIndyNamespace is required when using the Indy VDR',
-          schema: options.schema,
-          state: 'failed',
-        },
-      }
-    }
-
     try {
+      // This will throw an error if trying to register a schema with a legacy indy identifier. We only support did:indy identifiers
+      // for registering, that will allow us to extract the namespace and means all stored records will use did:indy identifiers.
+      const { id: unqualifiedDid, namespace } = parseIndyDid(options.schema.issuerId)
```

Comment:
This leaves me the same question about difference between `unqualifiedDid` and `didIdentifier` (or my suggested `namespaceIdentifier`)